source/server/converters.h

```diff
@@ -317,6 +318,19 @@ typename TypeToStorageType<TDriverType, TGrpcType>::StorageType allocate_output_
 }
 
 }  // namespace converters
+
+const int kMaxGrpcErrorDescriptionSize = 4096;
+
+inline ::grpc::Status ApiErrorAndDescriptionToStatus(google::protobuf::int32 status, std::string& description)
+{
+    converters::trim_trailing_nulls(description);
+    // TODO: May need full-fledged json escape logic.
```

Comment:
It also ought to convert the string encoding to UTF-8, but that looks like a much larger issue than just error reporting. 

The string encoding may vary by API. Most are ANSI. DAQmx has a separate UTF-8 DLL which this server is not using.

Reply:
BTW, I don't expect this to be addressed in this PR.

@astarche @reckenro Is the string encoding already on your radar or should I file a bug/issue?

Reply:
Good catch. I don't remember it being brought up before or tracked anywhere so I'd say filing a bug/issue sounds appropriate.

Reply:
https://github.com/ni/grpc-device/issues/693